src/pages/Invoice/CreateInvoice.js

```diff
@@ -1,7 +1,376 @@
-import React from "react";
+/* eslint-disable no-unused-vars */
+import React, { Fragment, useState, useEffect } from "react";
+import withWrapper from "common/withWrapper";
+import Navbar from "common/Navbar";
+import "styles/add-invoice.css";
+import { Link, useHistory } from "react-router-dom";
+import ChangeCustomerModal from "./ChangeCustomerModal";
+import AddItemModal from "./AddItemsModal";
+import { FormattedMessage } from "react-intl";
+import { useNotification } from "notification";
 
 const CreateInvoice = () => {
-    return <div>I am add customers page</div>;
+    const [itemModal, setItemModal] = useState(false);
+    const [customerDetailsModal, setCustomersDetailsModal] = useState(false);
```

Comment:
merge to one 
useState({ isOpen: false, type: 'customer' | 'item' })

Reply:
Done